metpy/plots/_util.py

```diff
@@ -50,7 +55,7 @@ def add_timestamp(ax, time=None, x=0.99, y=-0.04, ha='right', high_contrast=Fals
     text_args.update(**kwargs)
     if not time:
         time = datetime.utcnow()
-    timestr = datetime.strftime(time, 'Created: %Y-%m-%dT%H:%M:%SZ')
+    timestr = datetime.strftime(time, '{}{}'.format(pretext, time_format))
```

Comment:
This gives me some heartburn to require that people passing in their own `pretext` have to white space right pad the text and also ensure that their string does not contain `datetime.strftime` passable values.  What about
```
timestr = " ".join([pretext, datetime.strftime(time, time_format)])
```

Reply:
I'm not so sure I want to force the space - someone would have an application where they want to space (or just no pretext)n as soon as we do. I do agree with the point of making sure we separate the date time formatting and the pretext.

Reply:
We definitely should be doing the `strftime` separately. As far as the space is concerned, though, I'm inclined to use whatever the user passes in *as is*. We don't know that all of our users will want that space, or they may want *more* space. That means they're adding (N-1) spaces to account for ours. There's also the case where the user passes in `''` to only print the time, no text prefix.

Then there's the possibility that they actually want something like:
```
Created by Ryan
2017-05-17 12:57
```
That would require passing in `prefix="Created by Ryan\n"`, but then we would add a space after the `\n`, putting an extra space on the date time string; depending on the alignment settings, that would be bad.

So I guess what I'm saying is, let's invoke the zen of Python: "In the face of ambiguity, resist the temptation to guess."; no space--what you pass is exactly what you get.

Reply:
API design is hard.

Reply:
And we still got it wrong the first time 😄 - maybe the second 😈